tests/unit/runtime/zero/test_zero_context.py

```diff
@@ -263,8 +263,8 @@ def forward(self, input):
             # forward the external param
             return out_obj.out, out_obj.bias
         else:
+
             out, bias = self.d_linear(out)
-            assert bias.ds_status == ZeroParamStatus.AVAILABLE
```

Comment:
Is something like this not needed?

```python
   assert hasattr(bias, 'ds_status') or hasattr(bias, 'ds_param_alias')
   z3_bias = bias if hasattr(bias, 'ds_status') else bias.ds_param_alias
   assert z3_bias.ds_status == ZeroParamStatus.AVAILABLE
```

Reply:
Oh great catch @tjruwase, we were so focused on the first part of the problem that forget to add back the status check :)